### ISSUE TITLE
Update rangy

### ISSUE THREAD

**Issue by petecleary** [fraywing/textAngular#549]
The version of rangy contains Range.detach() method which is no longer supported.
[https://dom.spec.whatwg.org/#dom-range-detach](https://dom.spec.whatwg.org/#dom-range-detach)
Chrome shows the warning inside the console.

Update the rangy component to be the latest version.
[https://github.com/timdown/rangy](https://github.com/timdown/rangy)

**Comment by SimeonC** [comment]
As I've said elsewhere, I'll update rangy to 1.3 when it actually get's out of it's alpha/beta/pre-release state.

If you want to avoid the message feel free to use the 1.3 versions, as far as I'm aware you shouldn't have any problems with it.
